### DISCUSSION TITLE
Has someone created a Windows/Mac desktop game? How is it? What are the limitations

### DISCUSSION THREAD

**Discussion by meiry** [axmolengine/axmol#1304] [Q&A] [answered]
Hello all,
As the topic says, can you share your experience with desktop development?
What should I be careful about, and what still needs to be developed in the engine for a desktop game?

**Comment by rh101** [comment, marked as answer]
Everything works just as well, if not better, than it did in Cocos2d-x. The v1.x version of Axmol is pretty stable, at least from what I've experienced, as most of my development is done with x64 Windows builds.  So, whatever you could do with Cocos2d-x, assuming you've used that before, then you should be able to do it with Axmol.

Is there something specific you need to know?
